addons/l10n_ch/models/account_invoice.py

```diff
@@ -80,15 +106,15 @@ def _space_isr_number(isr_number):
             isr_number = False
             isr_number_spaced = False
             if record.name and record.l10n_ch_isr_subscription:
-                invoice_issuer_ref = re.sub('^0*', '', record.l10n_ch_isr_subscription)
```

Comment:
Here subscription number is replaced by the extra field, but maybe that is because in our code, we had switched both fields. @yvaucher

Reply:
@jco-odoo @fclementic2c 

l10n_ch_postal contains a postal account number **10**-123-1 or a supplier subscription number **01**-123-1 (01 and 03 are for subscription numbers, other are e.g 10 are for postal account number. Both are supposed to disapear in 2020. And be replaced by IBAN and QR-IBAN.

We use the fields

`l10n_ch_postal` (external partner bank account)
`l10n_ch_subscription_chf` (my company bank account)
`l10n_ch_subscription_eur` (my company bank account)

To pay we use l10n_ch_postal
To receive money we use l10n_ch_subscription_xxx

In practice it is really rare to need EUR and you probably don't need to fill both on your company settings. Thus we could only have 1 field. Nevertheless, for configuration purpose it's simpler for the user to see them separately on the bank view, this could be solved by a different view on company's bank account.


`l10n_ch_isrb_customer_id` is a different type of information.

It could be added in a second module if needed, as it is used only if the financial institution is a bank.
If you are a customer at Postfinance you don't need it.

Reply:
@yvaucher @fclementic2c 

I suppose you want to put l10n_ch_isrb_customer_id in field number 4 of the specification you put at the top of this pr.  For this, however, we reuse the field l10n_ch_postal at the moment.  It is a bit weird, but it can work this way as we know the IBAN number for the company accounts normally, so it is not set by anything else (and only needed to pay a vendor normally).  I want to be sure you agree with me on this one already.